docs/src/basics/solve.md

```diff
@@ -14,8 +14,8 @@ solve(prob::SciMLBase.NonlinearProblem, args...; kwargs...)
 ## Iteration Controls
 
   - `maxiters::Int`: The maximum number of iterations to perform. Defaults to `1000`.
-  - `abstol::Number`: The absolute tolerance.
-  - `reltol::Number`: The relative tolerance.
+  - `abstol::Number`: The absolute tolerance. Defaults to `real(oneunit(T)) * (eps(real(one(T))))^(4 // 5)`.
+  - `reltol::Number`: The relative tolerance. Defaults to `real(oneunit(T)) * (eps(real(one(T))))^(2 // 5)`.
```

Comment:
While now we are using ```_get_tolerance``` from DiffEqBase.jl, shouldn't both ```reltol``` and ```abstol``` are ```real(oneunit(T)) * (eps(real(one(T))))^(4 // 5)```?

Reply:
See:
https://github.com/SciML/DiffEqBase.jl/blob/898e6bf252dbd3e4a0561b19f2015b9d6e0a8ffb/src/termination_conditions.jl#L219-L226

Reply:
Oops yeah typo